### PR TITLE
Cultists gain increased blood regeneration speed

### PR DESCRIPTION
**Pull request by BroseidonMOO**
:cl: grimreaperx15
add: Blood Cultists will now regenerate blood passively much faster
/:cl:

[]: # For a cult based around blood, nothing really cares about blood. Hopefully rituals will use blood soon, and it only makes sense for cultists to get a passive regeneration buff for it.

### CODE REVIEW

File: code/game/gamemodes/cult/cult.dm [hunk 1]
```diff
@@ -161,6 +161,7 @@
 	if(cult_mind.current.gain_antag_datum(/datum/antagonist/cultist))
 		if(stun)
 			cult_mind.current.Paralyse(5)
+		cult_mind.current.apply_status_effect(STATUS_EFFECT_CULT_BUFF)
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/cult.dm]: You want to do this in the antag datum itself.

**Comment by ma44** [comment]
When can i meta this to know who's a cultist?

**Comment by AdamElTablawy** [comment]
why are you buffing something for something that might not be added

**Comment by BroseidonMOO** [comment]
I mean, I guess you could meta it by beating the shit out of all the crew and seeing who gets their blood back a bit faster, but I think searching them for a tome would be a lot more reliable?

And I'm buffing it since according to style guidelines, everything is supposed to be a separate PR.

**Comment by Shapsy** [comment]
Why can't they just steal blood from medbay or eat food to regain blood faster or eat iron to regain blood faster

**Comment by BroseidonMOO** [comment]
Yeah, I'm gonna be honest here. I didn't really think this through. Thanks for pointing it out.

**Pull request closed without merging**